app/src/features/projects/CreateProjectPage.tsx

```diff
@@ -478,8 +480,20 @@ const CreateProjectPage: React.FC = () => {
     return <CircularProgress />;
   }
 
+  const handleLocationChange = (location: History.Location, action: History.Action) => {
```

Comment:
can we add a test for this function? simulate the back button click and test?

Reply:
Should be covered by the existing cancel tests now, as all of the various ways to leave the page should trigger the prompt.